packages/components/src/VirtualizedList/CellActions/CellActions.scss

```diff
@@ -1,17 +1,32 @@
 .tc-list-actions {
 	button {
-		height: 20px;
-		line-height: inherit;
 		opacity: 0;
-		padding-left: 0;
 	}
 }
 
 // manage actions display on row hover
-
 :global(.tc-list-large-row) .tc-list-actions :global(.tc-actions) > button,
 :global(.ReactVirtualized__Table__row) .tc-list-actions :global(.tc-actions) > button:focus,
 :global(.ReactVirtualized__Table__row):hover .tc-list-actions :global(.tc-actions) > button {
 	opacity: 1;
 	transition: opacity 0.15s ease-in;
 }
+
+// Large display specificities
+:global(.tc-list-large-row) .tc-list-actions :global(.tc-actions) > button {
```

Comment:
I agree with @jmfrancois, this is dangerous, we have to add more classes to our produced HTML to avoid this kind of selectors.

Why did you set it here and not in list style?